modules/nixcord/hm.nix

```
@@ -0,0 +1,18 @@
+{ config, lib, ... }:
+let
+  themeFile = config.lib.stylix.colors {
+    template = ../vesktop/template.mustache; # Uses the same template as vesktop
```

This comment might be redundant:

```suggestion
    template = ../vesktop/template.mustache;
```